README.md

```diff
@@ -123,7 +160,51 @@ git clone https://github.com/peng-lab/PyBaSiC.git
 For mitosis movie frames, we give PyBaSiC 2 frames before/after the desired frame (depending on position in movie) to perform illumination correction.
 We discuss different methods of illumination correction in [#1](https://github.com/WayScience/IDR_stream/issues/1).
 
-### DeepProfiler Feature Extractor:
+# CellProfiler Project Setup:
+
+CellProfiler will be used for both segmentation and feature extraction.
+For segmentation, we recommend using the Cellpose plugin for CellProfiler for projects centered around nuclei. 
+Cellpose and CellProfiler are already installed from source within the conda environment from the `Necessary Packages` section.
+
+To run the CellProfiler Cellpose plugin, the steps are as follows:
+
+#### Step 1: Clone CellProfiler Github
+
+We recommend putting it into a folder on your Desktop called `GitHub` for better organization and tracking of the repo.
+
+```console
+# Make sure to `cd` into the directory that you want the repo in (e.g. \~/Desktop/Github)
```

Comment:
```suggestion
# Make sure to `cd` into the directory that you want the repo in (e.g. ~/Desktop/Github)
```
I don't think the backslash is necessary for this absolute path.

Reply:
Agreed, don't know why it is there lol. Fixed!